roles/console-dso/templates/values/10-proxy.j2

```diff
@@ -1,4 +1,4 @@
-{% if dsc.proxy.enabled %}
+{% if dsc.proxy.enabled and dsc.global.offline != true %}
```

Comment:
Il y a quelque chose à mettre dans le no_proxy de la conf dsc pour éviter le crashloopback du pod server console : `tunneling socket could not be established, statusCode=503`. Mais je n'ai pas réussi à trouver.

Reply:
Ah bon ? C'est étonnant, si le serveur communique via les urls publiques il faut passer par le proxy et si il utilise directement les services Kubernetes alors le `no_proxy` devrait déjà faire l'affaire non ? (il faudrait qu'on travaille d'ailleurs sur l'utilisation des services Kube en direct pour gagner en performances notamment, on peut en discuter prochainement si tu veux)

Reply:
Avec l'aide d'Arnaud, on avait obtenu des logs avec `NODE_DEBUG: net,tls,http,https,stream,agent` qui nous avaient permis de pointer sur quelque chose de manquant dans le no_proxy mais je n'avais pas réussi à déterminer ce qu'il fallait rajouter à partir des logs.
Oui, c'est une excellente idée pour l'utilisation des services Kube en direct autant que possible !